Doc/howto/clinic.rst

```diff
@@ -1895,14 +1895,15 @@ blocks embedded in Python files look slightly different.  They look like this:
     #/*[python checksum:...]*/
 
 
-How to deprecate positional use of optional parameters
-------------------------------------------------------
+How to deprecate passing parameters positionally
+------------------------------------------------
 
 Argument Clinic provides syntax that makes it possible to generate code that
-deprecates positional use of optional parameters.
-For example, say we've got a module level function :py:func:`!foo.myfunc` that
-takes three :class:`int` parameters:
-optional parameters *a* and *b*, and keyword-only parameter *c*::
+deprecates passing :term:`arguments <argument>` positionally.
+For example, say we've got a module-level function :py:func:`!foo.myfunc`
+that has three :term:`parameters <parameter>`:
+positional-or-keyword parameters *a* and *b*,
+and a :ref:`keyword-only <keyword-only_parameter>` parameter *c*::
```

Comment:
Since there is a link to `parameter` in the same sentence, I do not think that a link to `keyword-only_parameter` is needed. They lead to the same glossary entry, why contains also the definition of positional-or-keyword parameters.

Reply:
Thanks. I'll get to this later today.